### PR TITLE
add Qwoyn Blockchain

### PR DESCRIPTION
**Pull request by dpdanpittman**

### CODE REVIEW

File: qwoyn/chain.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyn",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://qwoyn.studio/",
+  "pretty_name": "Qwoyn",
+  "chain_id": "higgs-boson-1",
+  "bech32_prefix": "qwoyn",
```
Comment (JeremyParish69) [inline qwoyn/chain.json:9]: Please register this prefix to slip173:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: qwoyn/chain.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyn",
```
Comment (JeremyParish69) [inline qwoyn/chain.json:3]: if this is a testnet, please rename the folder and the chain_name in both files to qwoyntestnet, and move the folder into the /testnets/ folder

File: testnets/qwoyntestnet/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            "exponent": 6
+          }
+        ],
+        "base": "uqwoyn",
+        "name": "Qwoyn Blockchain",
+        "display": "qwoyn",
+        "symbol": "QWOYN",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/qwoyn/images/qwoyn.png"
+        },
```
Comment (JeremyParish69) [inline testnets/qwoyntestnet/assetlist.json:23]: extra comma here needs to go

File: testnets/qwoyntestnet/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "qwoyn",
```
Comment (JeremyParish69) [inline testnets/qwoyntestnet/assetlist.json:3]: this should be 'qwoyntestnet' too

**Comment by dpdanpittman** [comment]
Done

**Comment by dpdanpittman** [comment]
Not sure why the validate assetlist.json test is failing.  ` json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 24 column 7 (char 650)`

File: testnets/qwoyntestnet/chain.json [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  "website": "https://qwoyn.studio/",
+  "pretty_name": "Qwoyn",
+  "chain_id": "higgs-boson-1",
+  "bech32_prefix": "qwoyn",
+  "daemon_name": "qwoynd",
+  "node_home": "$HOME/.qwoynd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 123,
```
Comment (JeremyParish69) [inline testnets/qwoyntestnet/chain.json:15]: Slip44 123 is not registered to qwoyn. Please correct this. Was 118, not correct?

File: testnets/qwoyntestnet/chain.json [hunk 6]
```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyntestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://qwoyn.studio/",
+  "pretty_name": "Qwoyn",
+  "chain_id": "higgs-boson-1",
+  "bech32_prefix": "qwoyn",
```
Comment (JeremyParish69) [inline testnets/qwoyntestnet/chain.json:9]: Will be able to proceed once registered to that repo

File: qwoyn/chain.json [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  "status": "live",
+  "network_type": "testnet",
```
Comment (JeremyParish69) [inline qwoyn/chain.json:5]: Assuming you mean to register a testnet and mainnet at the same time, then this should be mainnet

File: qwoyn/chain.json [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+        "address": "http://137.220.37.9:1317",
+        "provider": "Qwoyn Studios"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "skynetexplorers",
+      "url": "https://www.skynetexplorers.com/qwoyn-studios",
+      "tx_page": "https://www.skynetexplorers.com/akash-network/txs/${txHash}",
```
Comment (JeremyParish69) [inline qwoyn/chain.json:83]: this says 'akash'?

File: testnets/qwoyntestnet/chain.json [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+        "address": "http://137.220.37.9:1317",
+        "provider": "Qwoyn Studios"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "skynetexplorers",
+      "url": "https://www.skynetexplorers.com/qwoyn-studios",
+      "tx_page": "https://www.skynetexplorers.com/akash-network/txs/${txHash}",
```
Comment (JeremyParish69) [inline testnets/qwoyntestnet/chain.json:83]: again, these pages say 'akash'

File: testnets/qwoyntestnet/chain.json [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+      {
+        "address": "http://137.220.37.9:1317",
+        "provider": "Qwoyn Studios"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "skynetexplorers",
+      "url": "https://www.skynetexplorers.com/qwoyn-studios",
```
Comment (JeremyParish69) [inline testnets/qwoyntestnet/chain.json:82]: url not found...

File: qwoyn/chain.json (hunk 7 above)
Comment (JeremyParish69) [reply qwoyn/chain.json:5]: based on other things I see, I don't think you meant to register a mainnet... so perhaps the mainnet files aught to be removed?

File: testnets/qwoyntestnet/chain.json (hunk 5 above)
Comment (dpdanpittman) [reply testnets/qwoyntestnet/chain.json:15]: That's right, 118 was not correct.

Comment (dpdanpittman) [reply testnets/qwoyntestnet/chain.json:15]: I'm confused though, I don't see any cosmos chains registered in slip44. I only see them in slip 173 and none of them are numbered.

File: testnets/qwoyntestnet/chain.json (hunk 10 above)
Comment (dpdanpittman) [reply testnets/qwoyntestnet/chain.json:82]: This will be launched today

File: testnets/qwoyntestnet/chain.json (hunk 5 above)
Comment (JeremyParish69) [reply testnets/qwoyntestnet/chain.json:15]: For SLIP44 (for address generation), the majority use the same method as Cosmos (118) and Ethereum (60)

File: testnets/qwoyntestnet/chain.json (hunk 6 above)
Comment (JeremyParish69) [reply testnets/qwoyntestnet/chain.json:9]: SLIP_0173 will have all the chains listed, with their bech32 prefix specified. No number is used.

**Comment by dpdanpittman** [comment]
Aha ok. Ill make the change

**Comment by dpdanpittman** [comment]
Everything should be fixed now.  Thanks for working with me on this @JeremyParish69

File: testnets/qwoyntestnet/chain.json (hunk 6 above)
Comment (dpdanpittman) [reply testnets/qwoyntestnet/chain.json:9]: Registered now

**Pull request merged**